resources/language/resource.language.es_es/strings.po

```diff
@@ -650,7 +654,7 @@ msgstr "Vista para Mi lista"
 
 msgctxt "#30166"
 msgid "Main menu items"
-msgstr "Eltos. del menú principal"
+msgstr "Elementos de menú"
```

Comment:
The 'main' adjective is missing in your translation ('principal' in spanish). Keep in mind there are submenus in the add-on so this adjective is informative. I had to abbreviate the word 'Elementos' to avoid text scrolling. But if you want to use this translation, at least replace 'de' with 'del' for a more natural translation.

Reply:
If by me it was I had put 'appearance' or only 'elements'

Reply:
> If by me it was I had put 'appearance/apariencia' or only 'elements/elementos'

But I think that the author wanted to be as precise as possible to make the function of the option perfectly clear. We should respect the author's text whenever possible and we only should change some words if the translated text is not natural (and always without modifying the meaning of the original statement).